src/views/search/SummaryUx.tsx

```diff
@@ -44,7 +44,13 @@ export const SummaryUx = () => {
       summary = unorderedSummary.replace(/\[\d+\]/g, "");
     }
   }
-
+  if(enableStreamQuery && unorderedSummary) {
+    processCitations(unorderedSummary)
```

Comment:
How does processing of citations (reordering, and replacing with links to URLS) happen in streaming?

Reply:
Its work as it was working earlier. Now in case of streaming whatever summary we have, we process citation on that.